Source/WebCore/css/MediaList.h

```diff
@@ -96,6 +96,7 @@ class MediaList final : public RefCounted<MediaList> {
     void clearParentStyleSheet() { ASSERT(m_parentStyleSheet); m_parentStyleSheet = nullptr; }
     void clearParentRule() { ASSERT(m_parentRule); m_parentRule = nullptr; }
     const MediaQuerySet* queries() const { return m_mediaQueries.get(); }
+    MediaQuerySet* queries() { return m_mediaQueries.get(); }
```

Comment:
Where is this needed?

Reply:
I needed it for this logic inside the CSSStyleSheet constructor:
```
    WTF::switchOn(WTFMove(options.media), [this](RefPtr<MediaList>&& mediaList) {
         if (auto* queries = mediaList->queries())
             setMediaQueries(*queries);
```